### PR TITLE
resolving merge conflicts

### PR DESCRIPTION
**Pull request by aatmanvaidya**

### CODE REVIEW

**Comment by aatmanvaidya** [comment]
just flagging an error that I got 
```sh
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: 'METADATA'
Check the permissions.
```

To fix this error we will have to do pip install as `user`

File: src/core/operators/audio_cnn_model/config.py [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
 
-labels_csv_path = '{}/panns_data/class_labels_indices.csv'.format(str(Path.home()))
+labels_csv_path = "{}/panns_data/class_labels_indices.csv".format(str(Path.home()))
 
 # Download labels if not exist
 if not os.path.isfile(labels_csv_path):
-	os.makedirs(os.path.dirname(labels_csv_path), exist_ok=True)
-	os.system('wget -O "{}" "http://storage.googleapis.com/us_audioset/youtube_corpus/v1/csv/class_labels_indices.csv"'.format(labels_csv_path))
+    os.makedirs(os.path.dirname(labels_csv_path), exist_ok=True)
+    os.system(
```
[the remark this answers is not in the document]
Comment (duggalsu) [reply src/core/operators/audio_cnn_model/config.py:14]: Will fix later

File: src/core/models/media_factory.py [hunk 2]
```diff
@@ -80,10 +87,11 @@
     @staticmethod
     def make_from_file_in_memory(file_data: FileStorage):
         # save on disk
-        fname = "/tmp/"+file_data.filename
+        fname = "/tmp/" + file_data.filename
```
[the remark this answers is not in the document]
Comment (duggalsu) [reply src/core/models/media_factory.py:90]: Will fix later

File: src/tests/core/models/test_media_factory.py [hunk 3]
```diff
@@ -17,43 +20,43 @@
...
+        self.assertTrue(isinstance(image_obj["image_array"], np.ndarray))
 
     # @skip
     def test_video_make_from_url(self):
         video_url = "https://tattle-media.s3.amazonaws.com/test-data/tattle-search/cat_vid_2mb.mp4"
         result = VideoFactory.make_from_url(video_url)
-        self.assertIsNotNone(result['path'])
-        self.assertEqual(result['path'], "/tmp/vid.mp4")
+        self.assertIsNotNone(result["path"])
+        self.assertEqual(result["path"], "/tmp/vid.mp4")
```
[the remark this answers is not in the document]
Comment (duggalsu) [reply src/tests/core/models/test_media_factory.py:40]: Will fix later

File: src/tests/core/models/test_media_factory.py [hunk 4]
```diff
@@ -17,43 +20,43 @@
...
 
     # @skip
     def test_audio_make_from_url(self):
         result = AudioFactory.make_from_url(
             "https://raw.githubusercontent.com/tattle-made/feluda/main/src/core/operators/sample_data/audio.wav"
         )
-        self.assertIsNotNone(result['path'])
-        self.assertEqual(result['path'], "/tmp/audio.wav")
+        self.assertIsNotNone(result["path"])
+        self.assertEqual(result["path"], "/tmp/audio.wav")
```
[the remark this answers is not in the document]
Comment (duggalsu) [reply src/tests/core/models/test_media_factory.py:55]: Will fix later

**Review by duggalsu — APPROVED** [review]

**Pull request merged**